101-VM-Virus-Attack-Prevention/azuredeploy.parameters.json

```diff
@@ -2,38 +2,15 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-      "location": {
-        "value": "East US"
-      },
-      "omsSku": {
-        "value": "PerNode"
-      },
-      "vNetName": {
-        "value": "GEN-VNET-NAME"
-      },
-      "vNetAddressSpace": {
-        "value": "10.1.0.0/16"
-      },
-      "subnetName": {
-        "value": "GEN-SUBNET1-NAME"
-      },
-      "subnetAddressSpace": {
-        "value": "10.1.0.0/24"
-      },
-      "nsgNamePrefix": {
-        "value": "virus-attack"
-      },
-      "pipAddressType": {
-        "value": "dynamic"
-      },
-      "vmNamePrefix": {
-        "value": "vm"
-      },
       "adminUserName": {
         "value": "VmUser"
       },
       "adminUserPassword": {
         "value": "GEN-PASSWORD"
+      },
+      "uniqueStringForPublicIP": {
```

Comment:
remove this since the param will be removed

Reply:
removed.